app/models/review.rb

```diff
@@ -10,11 +10,16 @@ class Review < ActiveRecord::Base
 
   belongs_to :venue
   belongs_to :user
+  has_many :votes
 
+  include PgSearch
+  pg_search_scope :search_by_review_body,
+    against: :review_body,
+    :using => {
+                :tsearch => {:prefix => true}
+              }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.